jupyterlab_vega/__init__.py

```diff
@@ -1,11 +1,9 @@
-from IPython.display import display
-from .utils import prepare_spec
+from IPython.display import display, DisplayObject
```

Comment:
I don't think we are using the `DisplayObject` so it can be removed from the import.